test/api/resources/accounts.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import APIResource from 'api/apiResource';
+import AccountResource from 'api/resources/accounts';
+
+describe('AccountResouce', () => {
+	const defaultBasePath = 'http://localhost:1234';
+	const defaultPath = '/accounts';
+
+	let LiskAPI;
+	let resource;
+
+	beforeEach(() => {
+		LiskAPI = {
+			headers: {},
+			fullURL: defaultBasePath,
+			hasAvailableNodes: () => {},
+			randomizeNodes: () => {},
+			banActiveNodeAndSelect: () => {},
+		};
+		resource = new AccountResource(LiskAPI);
+	});
+
+	describe('#constructor', () => {
+		it('should throw error without LiskAPI input', () => {
+			return (() => new AccountResource()).should.throw(
+				'Require LiskAPI instance to be initialized.',
```

Comment:
`AccountResource constructor requires a LiskAPI instance for initialization.`

Reply:
This behavior is defined in `APIResource`, so I am not sure if we should extend this just to add message

Reply:
This should apply to `APIResource`. My issue is with the wording, which is currently ungrammatical.